### PR TITLE
Fix linting, and add GitHub-actions

### PR DESCRIPTION
**Pull request by thaJeztah**
Follow-up to https://github.com/containerd/nri/pull/7

Project does not have tests yet, but enabling GHA shouldn't hurt

### CODE REVIEW

**Comment by thaJeztah** [comment]
Looks like it's working; https://github.com/thaJeztah/nri/pull/1 to see the results

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # nri - Node Resource Interface
 
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/containerd/nri)](https://pkg.go.dev/github.com/containerd/nri)
+[![Build Status](https://github.com/containerd/nri/workflows/CI/badge.svg)](https://github.com/containerd/nri/actions?query=workflow%3ACI)
+[![codecov](https://codecov.io/gh/containerd/aufs/branch/master/graph/badge.svg)](https://codecov.io/gh/containerd/nri)
```
Comment (thaJeztah) [inline README.md:5]: oh! need too fix this one; hold on

Comment (thaJeztah) [reply README.md]: fixed 👍

**Comment by thaJeztah** [comment]
@fuweid @AkihiroSuda @estesp ptal

(one more follow-up coming after this is merged)

**Review by AkihiroSuda — APPROVED** [review]

**Review by estesp — APPROVED** [review]
LGTM

**Pull request merged**